<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockPage</class>
 <widget class="QWidget" name="LockPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>登录密码:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>938</x>
     <y>10</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>287</y>
     <width>18</width>
     <height>7</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>263</y>
     <width>260</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border: 1px solid white;border-radius: 27px;padding: 1px 2px 1px 8px;min-width: 9em;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>360</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>916</x>
     <y>10</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>loginLabel</zorder>
  <zorder>closeBtn</zorder>
  <zorder>pwdLineEdit</zorder>
  <zorder>enterBtn</zorder>
  <zorder>tipLabel</zorder>
  <zorder>minBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
